css/portal-dashboard/num-students.less

```diff
@@ -0,0 +1,23 @@
+@import "./variables";
+
+.numStudentsContainer {
+    width: 168px;
+    height: 30px;
+    border-radius: 15px 0px 0px 15px;
+    background-color: @cc-teal-light7;
+    position: absolute;
+    top: 210px;
+    left: 72px;
+    font-size: 16px;
+    font-family: lato, arial, helvetica, sans-serif;
+    letter-spacing: 0;
+    color: @cc-charcoal;
+}
+.containerLabel {
+    font-weight: bold;
+    padding-left:20px;
+}
+.numStudentsContainer span {
+    position: relative;
+    top: 6px;
+}
```

Comment:
Add newline at end of file.  We should add this to our linter (we tend to in other projects).

Reply:
I already added it, but I don't think our `.less` files get linted currently.

Reply:
Ah, good point.  For now, we should be careful to do this by hand if the linter doesn't catch it.

Reply:
Looks like [lesshint](https://www.npmjs.com/package/lesshint) supports this as well as some other potentially useful checks.